maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -310,6 +343,57 @@ private void downloadOWAs(File targetDirectory, DistroProperties distroPropertie
 		}
 	}
 
+	/**
+	 * Sets the configuration folder for the specified server using the provided distro properties.
```

Comment:
Is the configuration folder specified in the distro properties file?

Reply:
I'll change this javadoc

Reply:
Configurations that are like config.referenceapplication-distro=3.0.0-SNAPSHOT. We don't specify the folder, just the version of configurations.